Content.Server/Xenoarchaeology/Equipment/ArtifactAnalyzerSystem.cs

```diff
@@ -84,7 +84,10 @@ private void UpdateClientUI(EntityUid console, ArtifactAnalyzerComponent analyze
     // DeltaV
     private float GetGlimmerMultiplier(ArtifactAnalyzerComponent comp)
     {
-        return 1 + (MathF.Pow(_glimmerSystem.Glimmer / 1000f, 2f) * comp.PointGlimmerMultiplier);
+        float normalizedGlimmer = _glimmerSystem.Glimmer / 1000f;
+        //DeltaV - Prevents extreme glimmer multipliers
+        return 1 + Math.Clamp((8f / 3f) * MathF.Pow(normalizedGlimmer, 3f) - 2f * MathF.Pow(normalizedGlimmer, 2f) + (4f / 3f) * (normalizedGlimmer), 0f, 2f);
```

Comment:
Welp, had to add two more terms, but it does the trick.

```suggestion
        return 0.5f + Math.Clamp(26 * MathF.Pow(normalizedGlimmer, 5) - 52 * MathF.Pow(normalizedGlimmer, 4) + 40 * MathF.Pow(normalizedGlimmer, 3) - 16 * MathF.Pow(normalizedGlimmer, 2) + 4.45f * normalizedGlimmer, 0, 2.5f);
```

I have not yet tested this in game, I'll do that now

Reply:
Oh wait just saw yours. Yours is way better.